infra/core/iam/subscription/port-forward/cluster-port-forward-user-role.bicep

```diff
@@ -0,0 +1,22 @@
+targetScope = 'subscription'
+
+@description('Name of the Custom Role Definition in Azure.')
+param roleName string
+
+@description('ID of the Access group.')
+param groupObjectId string
+
+@description('Required. Environment name.')
+param environment string
+
+var groupObjectIdvar = empty(groupObjectId) ? 'defaultgroupObjectIdforwhatif' : groupObjectId
+
+
+resource roleAssignmentAG 'Microsoft.Authorization/roleAssignments@2022-04-01' = if (environment == 'SND' || environment == 'DEV') {
```

Comment:
Can we make this an environment variable instead of hardcoding in the bicep?

Reply:
"defaultgroupObjectIdforwhatif " this is a dummy value, hence hardcoded.